lib/fetch_HLS.py

```diff
@@ -14,6 +15,26 @@
 from maap.maap import MAAP
 maap = MAAP()
 
+
+HLS_GEOPARQUET_BUCKET = "maap-ops-workspace"
+HLS_GEOPARQUET_KEY = "shared/henrydevseed/hls_boreal_20190701-20190831_20220701-20220831_20230701-20230831.parquet"
+
+
+def get_hls_geoparquet() -> str:
+    """Download the HLS STAC GeoParquet archive
+
+    The archive contains all HLSS30 and HLSL30 records for the bounding box (-180, 30, 180, 80)
+    for the time periods 2019-07-01 - 2019-08-31, 2022-07-01 - 2022-08-31, 2023-07-01 - 2023-08-31
+    """
+    aws_session = boto3.session.Session()
+    s3 = aws_session.resource("s3")
+
+    output_file = os.path.join("/tmp", os.path.basename(HLS_GEOPARQUET_KEY))
+    if not os.path.exists(output_file):
+        s3.Bucket(HLS_GEOPARQUET_BUCKET).download_file(HLS_GEOPARQUET_KEY, output_file)
```

Comment:
the download operation might not be necessary if the maap-ops-workspace bucket is available in some kind of storage mount.